events/wc_sacrifice_events.txt

```diff
@@ -0,0 +1,305 @@
+					}
+					else = { set_variable = { which = local_received_offmap_old_gods_currency_insanity_from_province_population_sacrifice_variable value = 600 } }
+				}
+				20 = {
+					if = {
+						limit = { any_owned_bloodline = { has_bloodline_flag = bloodline_insanity_gain } }
+						set_variable = { which = local_received_offmap_old_gods_currency_insanity_from_province_population_sacrifice_variable value = 600 }
+					}
+					else = { set_variable = { which = local_received_offmap_old_gods_currency_insanity_from_province_population_sacrifice_variable value = 400 } }
+				}
+				30 = {
+					trigger = { FROM = { is_richer_holding = no } }
+
+					if = {
+						limit = { any_owned_bloodline = { has_bloodline_flag = bloodline_insanity_gain } }
+						set_variable = { which = local_received_offmap_old_gods_currency_insanity_from_province_population_sacrifice_variable value = 300 }
+					}
+					else = { set_variable = { which = local_received_offmap_old_gods_currency_insanity_from_province_population_sacrifice_variable value = 200 } }
+				}
+				40 = {
+					trigger = { FROM = { is_rich_holding = no } }
+
+					if = {
+						limit = { any_owned_bloodline = { has_bloodline_flag = bloodline_insanity_gain } }
+						set_variable = { which = local_received_offmap_old_gods_currency_insanity_from_province_population_sacrifice_variable value = 120 }
+					}
+					else = { set_variable = { which = local_received_offmap_old_gods_currency_insanity_from_province_population_sacrifice_variable value = 80 } }
+				}
+			}
+		}
+		else_if = { # Or get sacrifice points
+			limit = { religion_have_ritual_sacrifice_trigger = yes }
+
+			bloodthirsty_tiered_traits_effect = yes
+		}
+
+		piety = 25
+
+		FROM = { # Sacrifice province population
+			add_holding_modifier = { name = inhabitants_sacrificed years = 5 }
+			if = {
+				limit = { settlement_can_be_destroyed_trigger = yes }
+				destroy_random_building = THIS
+				location = {
+					if = {
+						limit = { has_wonder = yes }
+						random = {
+							chance = 30
+							wealth = 100
+							wonder = { set_wonder_damaged = yes }
+						}
+					}
+				}
+			}
+
+			if = { # Pick character to sacrifice
+				limit = { is_capital = yes }
+				owner = {
+					random_courtier = {
+						limit = {
+							is_ruler = no
+							can_be_sacrificed_trigger = yes
+						}
+						random = {
+							chance = 25
+							modifier = {
+								factor = 3
+								is_unimportant_trigger = yes
+							}
+							modifier = {
+								factor = 3
+								has_minor_title = title_commander
+							}
+							modifier = { #Less likely to be in battle.
+								factor = 0.5
+								is_adult = no
+							}
+							modifier = { #Less likely to be in battle.
+								factor = 0.5
+								is_adult = no
+								is_child_of = PREV
+							}
+							ROOT = { PREV = { save_event_target_as = target_sacrificed_character } }
```

Comment:
`target_sacrificed_character` not sacrificed if you're not an Old God's follower. Is it intended?

Reply:
True, will change it then.

Reply:
Should be fine now